src/components/IssueLotto.js

```diff
@@ -0,0 +1,30 @@
+const Lotto = require('./Lotto');
+const Validation = require('./Validation');
+const { UTILS } = require('../constant/constant');
+const { Random } = require('@woowacourse/mission-utils');
+
+class IssueLotto {
+  static setLottoNumber(purchase) {
+    Validation.validatePurchase(purchase);
+    const results = [];
+    const count = IssueLotto.#drawLotto({ results, purchase });
+    return [results, count];
+  }
```

Comment:
반환하는 순서가 중요하지 않다면 비구조화 할당으로 반환하는 것도 좋아보입니다.
```suggestion
  static setLottoNumber(purchase) {
    Validation.validatePurchase(purchase);
    const results = [];
    const count = IssueLotto.#drawLotto({ results, purchase });
    return { results, count };
  }
  // count [, count] = setLottoNumber(purchase);
  // const { count } = setLottoNumber(purchase);
```

Reply:
그러네요! 나중에 배열 메서드나 고차 함수를 쓸 것 같아서 위와 같이 배열 비구조화 할당으로 작성하였는데 그렇지 않았으니 객체 방식의 비구조화 할당이 더 좋아보입니다. 좋은 지적 감사드립니다!😃